tools/CharmTraceReplacements.json

```diff
@@ -0,0 +1,25 @@
+{
+    "BasicReplace": {
+        "Actions::MutateApply": [["Actions::MutateApply<", ""], [">()", "()"]],
+        "EvolutionMetavars<grmhd::AnalyticData::KhInstability>": [
+            ["EvolutionMetavars<grmhd::AnalyticData::KhInstability>",
+             "Metavars"]],
+        "evolution::dg::Actions": [["evolution::dg::Actions", "dg"]],
+        "grmhd::ValenciaDivClean::subcell": [
+            ["grmhd::ValenciaDivClean::subcell", "subcell"]]
+    },
+    "RegexReplace": {
```

Comment:
Might I suggest also adding this to the default `RegexReplace`?

```json
  "mem_monitor::ProcessArray" : [
    ["mem_monitor::ProcessArray<DgElementArray.*\"",
     "mem_monitor::ProcessArray<DgElementArray\""]],
```

This particular template is long because it has the full DgElementArray action list. The other memory monitor templates are very short and this don't need any replacing.